.github/workflows/build-and-release-maven.yml

```diff
@@ -91,12 +114,24 @@ jobs:
       - uses: actions/checkout@v3
         with:
           persist-credentials: false
-      - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.36.0
+      - name: "Setup Java Build with specific configuration"
+        if: inputs.java-version && inputs.java-distribution
+        uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.36.0
+        with:
+          java-version: ${{ inputs.java-version}}
+          java-distribution: ${{ inputs.java-distribution}}
+      - name: "Setup Java Build with default configuration"
+        if: inputs.java-version == '' || inputs.java-distribution == ''
+        uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.36.0
       - uses: Alfresco/alfresco-build-tools/.github/actions/configure-git-author@v1.36.0
         with:
           username: ${{ secrets.BOT_GITHUB_USERNAME }}
           email: ${{ secrets.BOT_GITHUB_EMAIL }}
       - name: "Build"
         run: mvn -B -V install -DskipTests ${{ inputs.build-args }}
-      - name: "Release"
+      - name: "Release with specific version"
+        if: inputs.release-version && inputs.next-dev-version
```

Comment:
Why is this syntax different than the usual (missing the `!= ''` comparisons that are otherwise present in the other if conditions)?

Reply:
Sure, I will add a link to an example workflow which uses this workflow with the default values of those newly added params. Although, I have already tested it by changing my existing workflow and it is working fine. But I will add one for visibility and confirmation.

And regarding - _Why is this syntax different than the usual (missing the `!= ''` comparisons that are otherwise present in the other if conditions)?_
Actually, I checked that this was working in case the input parameters are non empty or set when I didn't mention `!= ''` for comparison explicitly. But if mentioning this explicitly improves the readability of the code, I am absolutely okay with this. Please share your thoughts, I will change it accordingly. 

Thank you.